### PR TITLE
Update BranchInfo.props for utf8string

### PR DESCRIPTION
**Pull request by joshfree**
to "alphautf8string" so it sorts before "preview1" and so it's under the nuget 20 character limit

### CODE REVIEW

File: BranchInfo.props [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
-    <PreReleaseLabel Condition="'$(PreReleaseLabel)' == ''">preview1</PreReleaseLabel>
+    <PreReleaseLabel Condition="'$(PreReleaseLabel)' == ''">alphautf8string</PreReleaseLabel>
     <IncludePreReleaseLabelInPackageVersion Condition="'$(StabilizePackageVersion)' != 'true' or '$(PackageVersionStamp)' != ''">true</IncludePreReleaseLabelInPackageVersion>
     <IncludeBuildNumberInPackageVersion Condition="'$(StabilizePackageVersion)' != 'true'">true</IncludeBuildNumberInPackageVersion>
 
     <ReleaseSuffix>$(PreReleaseLabel)</ReleaseSuffix>
-    <ReleaseBrandSuffix>Preview 1</ReleaseBrandSuffix>
+    <ReleaseBrandSuffix>Alpha Feature Utf8String</ReleaseBrandSuffix>
     <Channel>master</Channel>
     <BranchName>master</BranchName>
```
Comment (eerhardt) [inline BranchInfo.props]: @joshfree - You are going to want to update both the `Channel` and `BranchName` as well.

Comment (weshaggard) [reply BranchInfo.props]: @eerhardt what do those feed into? Is that for the azure blob storage location? If so I'm not sure we need to adjust them for this short-lived dev branch as there shouldn't be conflicts.

Comment (joshfree) [reply BranchInfo.props]: @eerhardt is Channel the dotnet/versions channel or something different?

Comment (eerhardt) [reply BranchInfo.props]: `$(Channel)` is used here:

https://github.com/dotnet/core-setup/blob/09c681b7a46163f9ef4e2fc41ca4ad3cba346734/publish/publish.proj#L146-L158

It is used to decide which "channel" to update the `latest` build for, which has links on the README.  For example:

https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-runtime-latest-win-x64.exe

If you don't update this, builds coming from this branch will show up as the latest `master` build on that link, which is definitely not what we want.

______________

`$(BranchName)` -  I thought this used to be used to update the `dotnet/versions` repo but we do that by using `$(SourceBranch)` in our build definitions:

https://github.com/dotnet/core-setup/search?utf8=%E2%9C%93&q=VersionsRepoPath&type=

I can't find any usages of `$(BranchName)` anymore.

Comment (weshaggard) [reply BranchInfo.props]: I wonder if we shouldn't use SourceBranch for the Channel by default as well and eliminate the need for changing these across branches.

**Pull request merged**